### ISSUE TITLE
Logo for DrWatson

### ISSUE THREAD

**Issue by cormullion** [JuliaDynamics/DrWatson.jl#20]
Extensive investigations revealed that the famous Dr Watson didn't wear a monocle... :)

![watson-banner-400](https://user-images.githubusercontent.com/52289/53792168-f4cca100-3f22-11e9-9c31-a37bee03353c.png)

**Comment by Datseris** [comment]
Awesomeeeee whats the font ?

**Comment by cormullion** [comment]
Top one is [Bernard Condensed](https://fontsinuse.com/typefaces/7791/bernard-condensed); lower one is Gill Sans.

**Comment by Datseris** [comment]
Veeeeeeery cool. So if you make a version without the dot I'll use it for our logo (should you allow it of course) . can you attach the icon as a separate file ? I guess it will have better resolution than me cropping it

**Comment by cormullion** [comment]
Source is here: https://gist.github.com/cormullion/bfd63e150b4fab45f69e8e77fbdb17e6 :)

I've put the SVGs here:

https://www.dropbox.com/s/txseo1aadjqqzd7/watson-banner.svg?dl=0

https://www.dropbox.com/s/nt6bnebzsro1b3i/dr-watson.svg?dl=0

Have fun! :)

**Comment by Datseris** [comment]
Thanks so much man. I only have one quick question: how do I modify the source code of the icon, so that the outskirts (the white corners just beyond the green and purple) are not white but transparent instead?

**Comment by Datseris** [comment]
Sorry for the spam, it is already transparent!

**Comment by Datseris** [comment]
![dancing script](https://user-images.githubusercontent.com/19669089/53806037-2312a680-3f4c-11e9-8edc-5c3fb41070e3.jpg)
![monsterrat](https://user-images.githubusercontent.com/19669089/53806038-2312a680-3f4c-11e9-9f3f-4377049e8f9d.jpg)

I've tried to change the small font into a script like font, as if Watson himself has written it or something. I also couldn't make Gill Sans look as nice as in your case so I went with Monsetrrat. Which one you think is better?
